packages/docs/content/migration-guides/introducing-a-css-reset.mdx

```diff
@@ -0,0 +1,57 @@
+
+Our reset CSS is bundled with the rest of the design system's styles, so to use it all you need to do is [import the design system's CSS](/getting-started/for-developers/#including-the-css) using your preferred method. Even if you haven't imported one of our components, your application should visually align with the design system.
+
+If you're curious to see what changes have been made, you can preview our reset styles in [Storybook](/storybook/?path=/story/reset-overview--overview). We'll break down the major changes for you to review below.
+
+## Global defaults
+
+The CSS reset introduces globally-defined defaults that will impact the entire application once the CSS. These defaults include (but aren't limited to):
+
+- `box-sizing: border-box` is set on every element — including `*::before` and `*::after`. This ensures that the declared width of element is never exceeded due to padding or border.
+- The `<html>` element has `scroll-behavior: smooth` set on `:focus-within`. This is a CSS-only solution that provides smooth anchor scrolling.
+- Animations, transitions, and smooth scroll are removed on all elements - including `*::before` and `*::after` for users who have `prefers-reduced-motion` enabled to respect their preferences.
+- The `<body>` element effectively replaces the `.ds-base` class, defining `font-family`, `font-size`, `line-height`, and `color`.
+  - In addition, `<body>` now sets `background-color` to a theme's specified background.
+- The `outline` is removed on elements with the `tabindex="-1"` declared. This is removes focus rings for non-interactive elements, e.g., when activating a skip link will then send focus to a `<main>` or heading element.
+- To avoid [margin collapse](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Model/Mastering_margin_collapsing), all elements will only rely on their respective top margins to better control layout. All bottom margins have been removed.
+
+## Typography
+
+### Headings
+
+All heading elements will receive the font styles defined by their parent element using the `font: inherit` rule. 
+
+[`font`](https://developer.mozilla.org/en-US/docs/Web/CSS/font) is a shorthand property that encompasses several font styles, but the key thing to note is a heading will now inherit the `font-family` and `font-size` of its parent or `<body>` element. This rule ensures the semantics of a heading element are distinct from its appearance. If you need to style a heading, refer to our [headings](/foundation/typography/headings/) or [utility class](/utilities/overview/) documentation.
+
+Lastly, headings have been reset to have their bottom margins removed. All headings have a top margin set to `1.5em`.
+
+### Paragraphs
+
+Like headings, paragraphs have had their bottom margins removed and a top margin of `1em` defined. However, if a paragraph is the `:first-child` its margin will be entirely removed.
```

Comment:
> However, if a paragraph is the `:first-child` its margin will be entirely removed.

Should we give the same treatment to headings? Has that been talked about?

Reply:
That's a good thought. Our heading style classes currently handle that behavior, but we could easily lift that to the HTML elements

Reply:
Should I update the reset to include the `:first-child`/`:last-child` rules from our [headings CSS](https://github.com/CMSgov/design-system/blob/main/packages/design-system/src/styles/_base.scss#L158)?

Reply:
Oh, I didn't know we were doing that! Yeah, I think moving that into the reset is a good idea, especially if the values don't change per heading level.